handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -107,87 +105,97 @@ static ServerContext newSessionContext(ReferenceCountedOpenSslContext thiz, long
                                            String keyPassword, KeyManagerFactory keyManagerFactory)
             throws SSLException {
         ServerContext result = new ServerContext();
+        OpenSslKeyMaterialProvider keyMaterialProvider = null;
         try {
-            SSLContext.setVerify(ctx, SSL.SSL_CVERIFY_NONE, VERIFY_DEPTH);
```

Comment:
note that most of the changes here is just because I added another level of `try {...} catch { ...}`